corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -447,6 +450,69 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
             }
         },
 
+        columnStyle: function () {
```

Comment:
I think this would be cleaner to do in HTML, with a `css` knockout bindings to apply classes conditionally, because styling info is usually in HTML/LESS rather than js. But this isn't a blocking comment.

Reply:
@orangejenny Do you have an example for that?

Reply:
Assuming you move the styles into two CSS classes and extract `self.showMap && !self.hasNoItems` into a `ko.computed`, you could then use `attr` to set the `class` like [this](https://github.com/dimagi/commcare-hq/blob/4c14b0b4e0e18d48a59810e4b14ebc4f463610fc/corehq/apps/reports/static/reports/js/single_form.js#L59), or use `css` like [this](https://github.com/dimagi/commcare-hq/blob/4c14b0b4e0e18d48a59810e4b14ebc4f463610fc/corehq/apps/cloudcare/templates/form_entry/templates.html#L46).